mirascope/core/openai/call_response.py

```diff
@@ -103,7 +103,7 @@ def cost(self) -> float | None:
     @property
     def message_param(self) -> SerializeAsAny[ChatCompletionAssistantMessageParam]:
         """Returns the assistants's response as a message parameter."""
-        return self.response.choices[0].message.model_dump(exclude={"function_call"})  # type: ignore
+        return self.response.choices[0].message.model_dump(exclude={"function_call"})  # pyright: ignore [reportReturnType]
```

Comment:
I wonder if we could do something like the following instead of ignoring:

```python
message_param = self.response.choices[0].message.model_dump(exclude={"function_call"})
return ChatCompletionAssistantMessageParam(**message_param)
```

Reply:
Thank you. I applied it.